src/GraphQL.Tests/Types/ComplexGraphTypeTests.cs

```diff
@@ -82,11 +90,16 @@ public void auto_register_object_graph_type()
         {
             GraphTypeTypeRegistry.Register<Money, AutoRegisteringObjectGraphType<Money>>();
 
-            var type = new AutoRegisteringObjectGraphType<TestObject>(o => o.valuePair, o => o.someEnumerable);
+            var type = new AutoRegisteringObjectGraphType<TestObject>(
+                true,
+                o => o.valuePair,
+                o => o.someEnumerable,
+                o => o.someIntMethod2(default, default, default)
```

Comment:
it looks a little strange

Reply:
Agree, but powerful to exclude overloaded methods